pkgs/by-name/ti/tidgi/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tidgi";
+  version = "0.8.0";
+
+  src =
+    if stdenv.isAarch64 then
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-arm64-${version}.zip";
+          hash = "sha256-e3ChSHljZSJiPh6KJ66BgAurBID/HiUQ7Gefq6mpU+c=";
+        }
+    else
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-x64-${version}.zip";
+          sha256 = "sha256-w3oNaiaThsjX8kBCUEgBMRVb+R57bH7zABM2K3bdUeU=";
+        };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ unzip ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Customizable personal knowledge-base with git as backup manager and blogging platform.";
```

```suggestion
    description = "Customizable personal knowledge-base and blogging platform with git as backup manager";
```
The description reads weirdly. I don't think git serves as the blogging platform, right?

---

I copy description from [Tidgi](https://github.com/tiddly-gittly/TidGi-Desktop)'s README.

---

Probably a translation mistake. In the same readme, it later says "It used to named TiddlyGit, means TiddlyWiki with easy Git backup, short for TidGi now."

---

I changed description.